docs/self-managed/zeebe-gateway-deployment/the-zeebe-gateway.md

```diff
@@ -25,9 +27,9 @@ To determine the current leader for the corresponding partition, the gateway mus
 
 The Zeebe Gateway protects the brokers from external sources. It allows the creation of a demilitarized zone ([DMZ](https://en.wikipedia.org/wiki/DMZ_(computing))) and the Zeebe Gateway is the only contact point.
 
-The Zeebe Gateway also allows you to easily create clients in your language of choice while keeping the client implementation as thin as possible. There are already several client implementations available, officially-supported, and community-maintained. Check the list [here](../../apis-clients/overview.md).
+The Zeebe Gateway also allows you to easily create clients in your language of choice while keeping the client implementation as thin as possible. There are already several client implementations available, officially-supported, and community-maintained. Check the list [here](../../apis-clients/overview.md). The clients can be kept thin, since the gateway takes care of the cluster topology and forwarding the requests to the right partitions.
```

Comment:
:wrench: If we're adding this note, then I would put it immediately after the first sentence, such that the last thing we mention is the list of clients. The ideas flow better and more naturally this way, since that last sentence is essentially clarifying what the first one said.

i.e.

```suggestion
The Zeebe Gateway also allows you to easily create clients in your language of choice while keeping the client implementation as thin as possible. The clients can be kept thin, since the gateway takes care of the cluster topology and forwarding the requests to the right partitions. There are already several client implementations available, officially-supported, and community-maintained. Check the list [here](../../apis-clients/overview.md).
```